cirq/experiments/qubit_characterizations_test.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+
+from cirq import GridQubit
+from cirq import circuits, ops, sim
+from cirq.experiments import rabi_oscillations, \
```

Comment:
Nit: use parens instead of `\`

Reply:
Done. Now the single-qubit clifford groups c_1, s_1 etc are all stored in a NamedTuple called Clifford.